src/main.ts

```diff
@@ -1,5 +1,7 @@
 import {app, BrowserWindow} from "electron";
 
+const EXCALIDRAW_BUNDLE = `file://${__dirname}/../excalidraw.asar/index.html`;
```

Comment:
I'm pretty sure we can not use this path on the distro build, that's why it's failing. Maybe I can have a look at it tomorrow

Reply:
Would be great!